iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/arithmetic.mlir

```diff
@@ -1,12 +1,122 @@
 // RUN: iree-opt -split-input-file -pass-pipeline='vm.module(iree-convert-vm-to-emitc)' %s | IreeFileCheck %s
 
-// CHECK: vm.module @module {
-vm.module @module {
+// CHECK: vm.module @my_module {
```

Comment:
Why does this one have a check for the module but none of the others do?

Reply:
For the added tests/newly tested ops, this is nearly a 1:1 copy from https://github.com/google/iree/blob/main/iree/compiler/Dialect/VM/IR/test/arithmetic_ops.mlir. These test cases do not contain checks for the module but our initial test case did (as well as for the return value). Testing those once is fine but I see no befit in testing this more than once. I hope that's okay (?).

Reply:
Yeah nbd either way. If it's useful to test, I probably wouldn't lump it in as part of the add test because it has nothing to do with add, per se

Reply:
Yeah, I agree. Will clean this up within the scope of some follow up work.